src/main/java/nextstep/courses/domain/session/Session.java

```diff
@@ -1,75 +1,117 @@
-        this.coverImg = coverImage;
+    public Session(CoverImages coverImages, Period period, SessionType sessionType) {
+        this.coverImages = coverImages;
         this.period = period;
         this.sessionType = sessionType;
-        this.course = course;
     }
 
-    public Session(Long id, SessionStatus status, LocalDateTime startDateTime, LocalDateTime endDateTime,
+    public Session(Long courseId, SessionStatus status, LocalDateTime startDateTime, LocalDateTime endDateTime,
+                   boolean free, Integer maxAttendance) {
+        this.courseId = courseId;
+        this.sessionStatus = status;
+        this.period = new Period(startDateTime, endDateTime);
+        this.sessionType = free ? SessionType.freeSession() : SessionType.notFreeSession(maxAttendance);
+    }
+
+    public Session(Long id, Long courseId, SessionStatus status, LocalDateTime startDateTime, LocalDateTime endDateTime,
                    boolean free, Integer maxAttendance) {
         this.id = id;
+        this.courseId = courseId;
         this.sessionStatus = status;
         this.period = new Period(startDateTime, endDateTime);
         this.sessionType = free ? SessionType.freeSession() : SessionType.notFreeSession(maxAttendance);
     }
 
-    public static Session notFreeSession(CoverImage coverImg, int maxAttendance, Course course, Period period) {
-        SessionType sessionType = SessionType.notFreeSession(maxAttendance);
-        Session session = new Session(coverImg, period, sessionType, course);
+    public Session(Long id, Long courseId, SessionStatus status, SessionEnrollStatus sessionEnrollStatus, LocalDateTime startDateTime, LocalDateTime endDateTime,
+                   boolean free, Integer maxAttendance) {
+        this.id = id;
+        this.courseId = courseId;
+        this.sessionStatus = status;
+        this.sessionEnrollStatus = sessionEnrollStatus;
+        this.period = new Period(startDateTime, endDateTime);
+        this.sessionType = free ? SessionType.freeSession() : SessionType.notFreeSession(maxAttendance);
+    }
 
-        course.addSession(session);
-        return session;
+    public static Session notFreeSession(CoverImages coverImages, int maxAttendance, Period period) {
+        SessionType sessionType = SessionType.notFreeSession(maxAttendance);
+        return new Session(coverImages, period, sessionType);
     }
 
-    public static Session freeSession(CoverImage coverImg, Course course, Period period) {
+    public static Session freeSession(CoverImages coverImages, Period period) {
         SessionType sessionType = SessionType.freeSession();
-        Session session = new Session(coverImg, period, sessionType, course);
+        return new Session(coverImages, period, sessionType);
+    }
+
+    public void addSessionUser(SessionUser sessionUser) {
+        if (sessionUser.userType() == UserType.TUTOR) {
```

Comment:
```suggestion
        if (sessionUser.isTutor()) {
```

값을 꺼내와서 비교하기 보다는 객체에게 비교하도록 시켜볼것 같아요!